autopush/src/settings.rs

```diff
@@ -73,8 +72,9 @@ impl Settings {
         s.set_default("max_connections", 0)?;
         s.set_default("close_handshake_timeout", 0)?;
         s.set_default("endpoint_scheme", "http")?;
+        s.set_default("endpoint_hostname", "localhost")?;
         s.set_default("endpoint_port", 8082)?;
-        s.set_default("crypto_key", Fernet::generate_key())?;
```

Comment:
I know that we're switching to a CSV list for this value.
Don't we reject any string that has "[]" as a wrapper in `ServerOptions::from_settings`?

Reply:
We reject the opposite: if the string does not have a `[]` wrapper.

Reply:
Ah! you're right. I missed the leading `!`.